test_runner/batch_others/test_tenant_tasks.py

```diff
@@ -0,0 +1,69 @@
+from fixtures.neon_fixtures import NeonEnvBuilder
+from uuid import UUID
+
+
+def get_only_element(l):
+    assert len(l) == 1
+    return l[0]
+
+
+def test_tenant_tasks(neon_env_builder: NeonEnvBuilder):
+    name = "test_tenant_tasks"
+    env = neon_env_builder.init_start()
+    client = env.pageserver.http_client()
+
+    def get_state(tenant):
+        all_states = client.tenant_list()
+        matching = [t for t in all_states if t["id"] == tenant.hex]
+        return get_only_element(matching)["state"]
+
+    def get_metric_value(name):
+        metrics = client.get_metrics()
+        relevant = [line for line in metrics.splitlines() if line.startswith(name)]
+        if len(relevant) == 0:
+            return 0
+        line = get_only_element(relevant)
+        value = line.lstrip(name).strip()
+        return int(value)
+
+    def detach_all_timelines(tenant):
+        timelines = [UUID(t["timeline_id"]) for t in client.timeline_list(tenant)]
+        for t in timelines:
+            client.timeline_detach(tenant, t)
+
+    # Create tenant, start compute
+    tenant, _ = env.neon_cli.create_tenant()
+    timeline = env.neon_cli.create_timeline(name, tenant_id=tenant)
+    pg = env.postgres.create_start(name, tenant_id=tenant)
+    assert (get_state(tenant) == "Active")
+
+    # Stop compute, detach timelines
+    # TODO tenant should go idle even if we don't explicitly detach
```

Comment:
I'll create an issue if this seems like desired behavior

Reply:
Please do. Tenant should eventually (when all the wal is consumed by pageserver) become idle when compute is down